addons/marketing_card/wizards/mail_compose_message.py

```diff
@@ -26,10 +26,18 @@ def _prepare_mail_values_dynamic(self, res_ids):
             ])
             for mail_values, body in zip(mail_values_all.values(), processed_bodies):
                 if body is not None:
+                    # in a mailing these are the same
+                    mail_values['body'] = body
                     mail_values['body_html'] = body
 
         return mail_values_all
 
+    def _get_done_emails(self, mail_values_dict):
```

Comment:
Which makes me think that the "done emails" computation on mailing is not aligned with "duplicate check". the "already sen" checks email, while the "currently being sent" checks all content. But not sure how we could solve it as we don't always keep sent content.

Reply:
Indeed, simplest way is probably to order by recipient email somehow to try and detect it while the email is being prepared in the composer. Other than that, a getter that says "emails from this mailing are not all the same". Or other non-trivial solutions (re-render, check dynamic values, ...) but I don't think we need to be super thorough.